pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -542,6 +546,27 @@ def _set_horovod_backend(self):
         else:
             self.num_processes = hvd.local_size()
 
+    def check_ipython_compatibility(self):
+        """Raises a `MisconfigurationException` if the accelerator is not compatible with IPython and code is run in an IPython kernel."""
+        if self._distrib_type not in self.ipython_compatible_distrib_types:
+            # check ipython env
+            import sys
```

Comment:
can you put the import at the top level?

Reply:
Since here is the only place where `sys` is used, I am hesitating on whether to move it up to the top level, but if you insist, it is fine for me.

Reply:
In Lightning it's always at the top no matter how many times it is used
cc @Borda

Reply:
yes, agree...

Reply:
OK, moved it